single-param-eval/prelim-single-param-eval.R

```diff
@@ -0,0 +1,98 @@
+# Title: Preliminary Single Parameter Tree Detection Evaluation
+# Description: This script uses statistics from run-tree-map-comparison.R (a
+#              script that compares predicted and observed tree maps) to summarize
+#              data contained in "stats_ ... .csv" files. It aims to identify an
+#              optimal parameter for two tree categories (i.e. 10+, overstory and
+#              10+/20+, all trees) based on the given parameter range.
+
+library(tidyverse)
+
+#### CONSTANTS ####
+STATS_DIR = "..." # path to folder containing match_stats.csv files
+OUT_DIR = "..." # path to save plot png
+
+PLOT_FILE_NAME = "...-plot" # file name to assign to plot png
+SINGLE_PARAM = "..." # the parameter of interest
+
```

Comment:
In a future version, this script could loop over all candidate parameters, rather than having to be run multiple times, each with manual hard-coded parameter names. Not something that needs to be changed now though.